### ISSUE TITLE
scaler transform in load_dataset function Causing speed data negative

### ISSUE THREAD

**Issue by HYTYH** [liyaguang/DCRNN#67]
In load_dataset function shown below:
![image](https://user-images.githubusercontent.com/53565598/128595887-f56314d2-a4c5-4012-b9eb-50f3f6f071ef.png)
Line 187 and 188, I'm confused these 2 lines may cause negative speed data.
And then I try to print the data in the dataloader in training process: (pytorch implement in dcrnn_supervisor.py)
![image](https://user-images.githubusercontent.com/53565598/128595945-27ea181a-1024-4354-a746-f7e3d39a8eea.png)
I found a lot of speed data is negative: (As shown below)
![image](https://user-images.githubusercontent.com/53565598/128595977-480ed789-4b49-472f-8207-a5a994545270.png)
The first column represent the speed and the second column is time.
Since loss function compute the mask after the scaler tranform operation, I think some of data input to the model are possibly wrong.
I hope to get answer to my question
